### PR TITLE
update certificates with expiration time 100 years

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
fixes: #13232

### CODE REVIEW

File: pilot/pkg/proxy/envoy/v2/init_test.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func connectADSS(url string) (ads.AggregatedDiscoveryService_StreamAggregatedRes
 	tlsCfg := &tls.Config{
 		Certificates: []tls.Certificate{clientCert},
 		RootCAs:      serverCAs,
-		ServerName:   "istio-pilot.istio-system.svc",
```
Comment (hzxuzhonghu) [inline pilot/pkg/proxy/envoy/v2/init_test.go]: I use citadel to generate the certificate, 
The SAN is 
```
 X509v3 Subject Alternative Name: 
                URI:spiffe://cluster.local/ns/istio-system/sa/istio-pilot-service-account, DNS:istio-pilot.istio-system
```

**Comment by hzxuzhonghu** [comment]
really have no idea why the codecov failed in `istio.io/istio/pilot/pkg/proxy/envoy/v2`, but the unit test can pass,

**Comment by hzxuzhonghu** [comment]
seems the codecov ci still test on previous commit,

https://circleci.com/gh/istio/istio/383192?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

It can never pass.

```
HEAD is now at dc2c4c835... Remove unused mixer.enabled value (#13214)
mkdir -p /go/out/logs
bin/testEnvLocalK8S.sh ensure
```

**Comment by howardjohn** [comment]
/lgtm 

a comment on what is needed to generate these would be nice but if its 100 years that is a problem for another generation.

**Pull request merged**